README.md

```diff
@@ -286,6 +286,8 @@ This plugin creates device plugin endpoints based on the configurations given in
 | "resourcePrefix"  | N        | Endpoint resource prefix name override. Should not contain special characters                                                          | string Default : "intel.com"                          | "yourcompany.com"                                                      |
 | "deviceType"      | N        | Device Type for a resource pool.                                                                                                       | string value of supported types. Default: "netDevice" | Currently supported values: "accelerator", "netDevice", "auxNetDevice" |
 | "excludeTopology" | N        | Exclude advertising of device's NUMA topology                                                                                          | bool Default: "false"                                 | "excludeTopology": true                                                |
+| "checkHealthOnPf" | N        | Check the health of a net device by inspecting the link state of the PF                                                                                       | bool Default: "false"                                 | "checkHealthOnPf": true                                                |
```

Comment:
should this be a list ? that way we dont overcumber the API 

e.g
`healthChecks: ["DeviceNetworkLink", DeviceExists]`